src/components/Card.js

```diff
@@ -6,75 +6,75 @@ export default class Card {
     handleDeleteClick,
     handleLikeClick
   ) {
-    this._name = data.name;
-    this._link = data.link;
-    this._id = data._id; // Keep track of card ID
-    this._cardSelector = cardSelector;
-    this._handleImageClick = handleImageClick;
-    this._handleDeleteClick = handleDeleteClick;
-    this._handleLikeClick = handleLikeClick;
```

Comment:
`_` is needed for the **private** methods. You should have removed it only from those, which you use in `index.js`